packages/atomic/cypress/integration/search-interface.cypress.ts

```diff
@@ -31,28 +29,57 @@ describe('Search Interface Component', () => {
     });
   };
 
-  it('should support changing language', () => {
-    setLanguage('fr');
-    cy.get('atomic-query-summary')
-      .shadow()
-      .invoke('text')
-      .should('contain', 'Résultats');
-
-    setLanguage('en');
-    cy.get('atomic-query-summary')
-      .shadow()
-      .invoke('text')
-      .should('contain', 'Results');
+  describe('with an automatic search', () => {
+    beforeEach(() => {
+      setUpPage('<atomic-query-summary>', true);
+    });
+
+    it('should support changing language', () => {
+      setLanguage('fr');
+      cy.get('atomic-query-summary')
+        .shadow()
+        .invoke('text')
+        .should('contain', 'Résultats');
+
+      setLanguage('en');
+      cy.get('atomic-query-summary')
+        .shadow()
+        .invoke('text')
+        .should('contain', 'Results');
+    });
+
+    it('should support changing a translation value without overriding other strings', () => {
+      setTranslation('fr', 'showingResultsOf_plural', 'patate');
+      setLanguage('fr');
+
+      cy.get('atomic-query-summary')
+        .shadow()
+        .invoke('text')
+        .should('contain', 'patate')
+        .should('contain', 'seconde');
+    });
   });
 
-  it('should support changing a translation value without overriding other strings', () => {
-    setTranslation('fr', 'showingResultsOf_plural', 'patate');
-    setLanguage('fr');
+  describe('without an automatic search', () => {
+    beforeEach(() => {
+      setUpPage('<atomic-query-summary>', false);
+    });
+
+    const execSearch = () =>
+      getSearchInterface(($si) => $si.executeFirstSearch());
 
-    cy.get('atomic-query-summary')
-      .shadow()
-      .invoke('text')
-      .should('contain', 'patate')
-      .should('contain', 'seconde');
+    it('should set locale for search request', async () => {
+      setLanguage('fr');
+      execSearch();
+      const req = await getApiRequestBodyAt('@coveoSearch', 0);
```

Comment:
Using async/await is not recommended with Cypress https://docs.cypress.io/faq/questions/using-cypress-faq#Can-I-use-the-new-ES7-async-await-syntax

Here's an alternative:
```
cy.wait(RouteAlias.analytics).then((intercept) => {
  const analyticsBody = intercept.request.body;
  expect(analyticsBody).to.have.property('language', 'fr');
});
```